src/components/ShadowedCard/ShadowedCard.js

```diff
@@ -120,6 +121,7 @@ class ShadowedCard extends React.Component<Props, State> {
       upperContentWrapperStyle = {},
       theme,
       isAnimated,
+      borderRadius,
```

Comment:
Since borderRadius can now be passed as prop, it should be also passed down to Shadow component ([L156](url)) to adjust Shadow's radius as well (not just the card's).
And it should be slightly smaller (by 2px) than the borderRadius that has been passed in.